action.yml

```diff
@@ -73,6 +73,13 @@ inputs:
       {{ custom }}
       {% endif %}
 
+      {% if backport %}
+      This release has been identified as a backport.
+      Automated changelogs for backports tend to be wildly incorrect.
+      Therefore, the list of issues and pull requests is hidden.
```

Comment:
I wonder if this is the best wording.
The ideal wording would have the following effects:
 - a normal user would be aware that this is a back-port, and not be too distracted by the stuff intended for maintainers
 - a maintainer would realize that they should edit this.

Possibly the maintainer would realize there is something filled in, before they click edit, but not necessarily, since it would be  obvious once they go to edit?